src/topics/group-generator/group-generator.test.ts

```diff
@@ -231,6 +232,31 @@ describe("test group generator", () => {
   });
 
   test("Should generate a group with the generator", async () => {
+    await service.generateGroups("test-generator,test-generator-2", {
```

Comment:
it seems that the title of this test has been exchanged with the name of the test below it no?